parsl/providers/ad_hoc/ad_hoc.py

```diff
@@ -255,3 +255,7 @@ def scaling_enabled(self):
     @property
     def label(self):
         return self._label
+
+    @property
+    def status_polling_interval(self):
```

Comment:
cmd_timeout: how long before commands used to interact with the scheduler are considered to have hanged when invoked
status_polling_interval: how frequently a specific command should be run in order to query the status of jobs)

Aren't those two completely unrelated things? On what basis would one ever assume that one can be derived from the other?

Reply:
They are unrelated until the polls hit the timeouts repeatedly. I guess this is fine since the polls are blocking, and we aren't going to add to the stress on a backed up batch-scheduler.